app/controllers/my_facilities_controller.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 class MyFacilitiesController < AdminController
-  include DistrictFiltering
```

Comment:
Why can we remove this?

Reply:
Its unused 🤷 
MyFacilities only has block level filtering in the UI, so this wasn't being used anywhere. It's also the only place that includes DistrictFiltering, removing this allows us to delete that concern as well.